package.json

```diff
@@ -53,15 +50,14 @@
     "console-browserify": "^1.2.0",
     "dayjs": "^1.10.8",
     "ethers": "^5.6.1",
-    "events": "^3.3.0",
+    "events": "^1.1.1",
```

Comment:
The dependencies that are automatically installed by `rn-nodify` are getting downgraded in the process. Those should not be removed in the first place.

Reply:
> should I keep the same downgraded versions?

No, please keep the versions as they currently are